src/Sidebar.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useState } from "react";
+import homeicon from "./Images/home-icon.png";
+import search from "./Images/search.png";
+import detail from "./Images/detail-icon.png";
+import baropener from "./Images/bar-opener.png";
+import profile from "./Images/profile logo.png";
+
+export default function Sidebar() {
+  const [sidebar, setsidebar] = useState(true);
+  const [colour, setcolour] = useState(`bg-gray-400`);
+  const [currentSelectedItem, setCurrentSelectedItem] = useState(null);
+  /* useEffect(()=>
+    {
+    },[currentSelectedItem]);
+*/
+  const controlsidebar = () => {
```

Comment:
use a better name for this. Camel casing should be followed too. For example, toogleCollapse